### ISSUE TITLE
Meanfile missing?

### ISSUE THREAD

**Issue by aghagol** [alexgkendall/caffe-posenet#4]
I can't seem to find the 'meanfile' file that is required in posenet/scripts/test_bayesian_posenet.py. Is that something left for us to compute or you would be so kind to share yours? :)

**Comment by alexgkendall** [comment]
You can easily compute it with `build/tools/compute_image_mean input_lmdb_database/ output.binaryproto`

See the caffe documentation for more details.

Thanks!

**Comment by aghagol** [comment]
Will do. Thanks for the quick reply.

**Comment by Soledad89** [comment]
@alexgkendall, Hi, I got the same error when using the command python test_posenet.py --model your_model.prototxt --weights your_weights.caffemodel --iter size_of_dataset, I got the output as below showing that mean file  could not be found, I'v computed the mean file using the caffe tools and I have put the mean file in the higher level directory of the test_posenet.py file:

I1024 16:38:49.718796   911 layer_factory.hpp:77] Creating layer data
I1024 16:38:49.719916   911 net.cpp:100] Creating Layer data
I1024 16:38:49.719925   911 net.cpp:408] data -> data
I1024 16:38:49.719934   911 net.cpp:408] data -> label
I1024 16:38:49.719944   911 data_transformer.cpp:25] Loading mean file from: ../imagemean.binaryproto
F1024 16:38:49.720598  1425 db_lmdb.hpp:15] Check failed: mdb_status == 0 (2 vs. 0) No such file or directory
**\* Check failure stack trace: ***

but I still got the error as above. I'm a new starter. Thank your for your reply.

**Comment by Henry-Faull** [comment]
I'm getting the exact same problem as @Soledad89 - I'm trying to run test_posenet.py and the imagemean.binaryproto file can't be found. It's been created and put in the directory above test_posenet.py. Was this ever resolved?

**Comment by DRAhmadFaraz** [comment]
@Henry-Faull @Soledad89  @aghagol  @alexgkendall 

Dear all, 

According to your instructions I have successfully installed Caffe from your repository.
When I Run this command "Python create_posenet_lmdb_dataset.py" It shiows like this.

![screenshot from 2019-01-28 10-09-06](https://user-images.githubusercontent.com/38114191/51829107-d0a6f000-22e4-11e9-9d2b-f3171e491d65.png)

After that When i use this command "python test_posenet.py --model train_kingscollege.prototxt --weights weights_kingscollege.caffemodel --iter 100" I got error of some mean file missings as shown.

![screenshot from 2019-01-28 10-11-08](https://user-images.githubusercontent.com/38114191/51829263-195ea900-22e5-11e9-8f0e-92dd149f00e0.png)

I0128 10:10:24.872309 20176 layer_factory.hpp:77] Creating layer data
I0128 10:10:24.872444 20176 net.cpp:106] Creating Layer data
I0128 10:10:24.872455 20176 net.cpp:411] data -> data
I0128 10:10:24.872473 20176 net.cpp:411] data -> label
I0128 10:10:24.872486 20176 data_transformer.cpp:25] Loading mean file from: ../imagemean.binaryproto
F0128 10:10:24.872498 20184 db_lmdb.hpp:14] Check failed: mdb_status == 0 (2 vs. 0) No such file or directory
*** Check failure stack trace: ***
F0128 10:10:24.872499 20176 io.cpp:54] Check failed: fd != -1 (-1 vs. -1) File not found: ../imagemean.binaryproto
*** Check failure stack trace: ***
Aborted (core dumped)

Kindly help me. I ll be thankful to you.
